File: fizzBuzz.py
import time
start_time = time.time()

def fizz_buzz():
    for i in range(0, 1000000):
        if i % 3 == 0 and i % 5 == 0:
            print('FizzBuzz')
        elif i % 3 == 0:
            print('Fizz')
        elif i % 5 == 0:
            print('Buzz')
        else:
            print(i)

fizz_buzz()
print("--- %s seconds ---" % (time.time() - start_time))

File: fizzBuzz.js
function fizzBuzz() {
  console.time("fizzBuzz");
  for (let i = 0; i <= 1000000; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
      console.log("FizzBuzz");
    } else if (i % 3 === 0) {
      console.log("Fizz");
    } else if (i % 5 === 0) {
      console.log("Buzz");
    } else {
      console.log(i);
    }
  }
  console.timeEnd("fizzBuzz");
}
fizzBuzz();
